### PR TITLE
Paul/docs/tweaks

### PR DESCRIPTION
**Pull request by yoid2000**

### CODE REVIEW

File: docs/operation.md [hunk 1]
```diff
@@ -189,43 +190,52 @@ The Anonymization Summary has four statistics:
 
 ### Example 1: Almost no distortion
 
-![](images/distortion-none.png#720)
+| Suppressed Count | Suppressed Bins | Median Distortion | Maximum Distortion |
+| --- | --- | --- | --- |
+| 0 of 440257 (0%) | 0 of 3 (0%) | 0.05% | 0.69% |
 
-The above image illustrates the case when there is almost no distortion. No data is suppressed, the median distortion is a small fraction of 1%, and even the maximum distortion is below 1%. Low distortion occurs where relative few output bins, and each bin contains data for a substantial number of protected entities.
+In this case is almost no distortion. No data is suppressed, the median distortion is a small fraction of 1%, and even the maximum distortion is below 1%. Low distortion occurs where there are relatively few output bins, and each bin contains data for a substantial number of protected entities.
```
Comment (edongashi) [inline docs/operation.md:197]: > In this case is almost no distortion.

Doesn't sound quite right.

> Low distortion occurs **where** there are relatively few output bins,

Where or when?

File: docs/operation.md [hunk 2]
```diff
@@ -189,43 +190,52 @@ The Anonymization Summary has four statistics:
...
 
 ### Example 3: Bad data quality
 
-![](images/distortion-bad.png#720)
+| Suppressed Count | Suppressed Bins | Median Distortion | Maximum Distortion |
+| --- | --- | --- | --- |
+| 398784 of 440257 (90.58%) | 11933 of 12995 (91.83%) | 94.87% | 97.06% |
 
-The above image illustrates the case when almost all of the data resides in very small bins. 91% of the data is suppressed (`Suppressed Count`), as is 92% of the bins. The median and maximum distortion (relative error) is nearly the same, and both are high (95% and 97%).
+In this case almost all of the data resides in very small bins. 91% of the data is suppressed (`Suppressed Count`), as is 92% of the bins. The median and maximum distortion (relative error) are both high (95% and 97%).
```
Comment (edongashi) [inline docs/operation.md:215]: In this case, <<< comma for consistency with above.

File: docs/operation.md [hunk 3]
```diff
@@ -189,43 +190,52 @@ The Anonymization Summary has four statistics:
...
 
 The `Combined view` additionally shows the original data side-by-side with the anonymized data. Because the combined view contains original data, it is not safe to release. The combined view presents two additional columns, `Count (original)` and `Distortion`. `Count (original)` is the true count of the original data. `Distortion` is the relative error between the original count and the anonymized count.
 
 The combined view displays suppressed bins. The `Count (anonymized)` column is left blank for suppressed bins.
 
 The combined view lets you examine precisely the distortion and suppression. The anonymized data can be sorted by each column. Sorting by `Count (original)` shows the ascending suppressed bins first, and so is useful for examining what has been suppressed. Sorting by `Distortion` descending shows the bins with the most error first, and so is useful for examining which bins have high distortion.
 
+_Note that the output view displays at most 1000 rows of data._ To view all of the data, use
+the CSV export (Anonymized view only).
```
Comment (edongashi) [inline docs/operation.md:228]: > To view all of the data, use the CSV export (Anonymized view only).

This makes you think that you can only export while in anonymized view. Maybe say (exports anonymized view only)?

**Comment by edongashi** [comment]
Done with review. The build is failing because the `notebook-help.tsx` file must be reformatted. I can push a fix after you handle the review comments.

**Comment by yoid2000** [comment]
@edongashi ok thanks. My changes done.

**Review by edongashi — APPROVED** [review]

**Pull request merged**